handbook/engineering/README.md

```diff
@@ -144,12 +144,11 @@ Engineering-initiated stories follow the [user story drafting process](https://f
 
 ### Creating an engineering-initiated story
 
-1. Create a [new user story issue](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=story%2C%3Aproduct&projects=&template=story.md&title=) in GitHub. 
+1. Create a [new feature request issue](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=~customer+request&projects=&template=feature-request.md&title=) in GitHub. 
```

Comment:
If we want Fleeties to use this template, should we remove the `!customer request` label? 

![image](https://github.com/fleetdm/fleet/assets/2495927/4e6cad1e-14ab-47d5-9389-6bc2488cb011)

Reply:
hmmmmm good point. Maybe it should actually be a separate template.

Reply:
Because when these are made and they inevitably de-prioritized, I'd love to have some way to look at all of these.

Reply:
```suggestion
1. Create an [engineer-initiated request issue](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=~engineering-initiated&projects=&template=eng-initiated-request.md&title=) in GitHub. 
```